crates/biome_configuration/src/javascript/formatter.rs

```diff
@@ -94,7 +94,10 @@ pub struct JsFormatterConfiguration {
 
     // it's also a top-level configurable property.
     /// The attribute position style in jsx elements. Defaults to auto.
-    #[bpaf(long("javascript-attribute-position"), argument("multiline|auto"))]
+    #[bpaf(
```

Comment:
Nit: While we’re at it, maybe we can capitalize JSX in the comment above 😅

Reply:
Fixed in [ddaa142](https://github.com/biomejs/biome/pull/5132/commits/ddaa1424a2f3eeaf2e2e7091e6552bab8037870a)!